src/PageTransition.js

```diff
@@ -262,6 +261,11 @@ PageTransition.propTypes = {
     if (props.loadingTimeout) pt = pt.isRequired
     return pt(props, ...args)
   },
+  timeout: (props, ...args) => {
+    let pt = timeoutsShape
+    if (pt) pt = pt.isRequired
+    return pt(props, ...args)
```

Comment:
pt could be `undefined`
```suggestion
    if (pt) {
      pt = pt.isRequired
      return pt(props, ...args)
    }
```

Reply:
Yep, agreed that this is the right thing to do.